spec/index.bs

```diff
@@ -1537,40 +1537,54 @@ interface GPUMapMode {
 ### <dfn method for=GPUBuffer>getMappedRange(offset, size)</dfn> ### {#GPUBuffer-getMappedRange}
 
 <div algorithm="GPUBuffer.getMappedRange">
-  <strong>|this|:</strong> of type {{GPUBuffer}}.
+    <strong>|this|:</strong> of type {{GPUBuffer}}.
+
+    **Arguments:**
 
-  **Arguments:**
     - {{GPUSize64}} |offset|
     - {{GPUSize64}} |size|
 
-  **Returns:** {{ArrayBuffer}}
+    **Returns:** {{ArrayBuffer}}
 
-  1. If this call doesn't follow the [$getMappedRange Valid Usage$]:
+    1. If |size| is unspecified:
 
-    1. Throw an {{OperationError}}.
+        - If |offset| &ge; this.{{[[size]]}}, throw an {{OperationError}} and stop.
+        - Let |rangeSize| be this.{{[[size]]}} - |offset|.
 
-  1. Let |m| be a new {{ArrayBuffer}} of size |size| pointing at the content of |this|.{{[[mapping]]}} at offset |offset| - |this|.{{[[mapping_range]]}}[0].
-  1. Append |m| to |this|.{{[[mapped_ranges]]}}.
-  1. Return |m|.
+        Otherwise, let |rangeSize| be |size|.
 
-  <div algorithm class=validusage>
-    <dfn abstract-op>getMappedRange Valid Usage</dfn>
+    1. If this call doesn't follow the [$getMappedRange Valid Usage$]:
 
-      Given a {{GPUBuffer}} |this|, a {{GPUSize64}} |offset| and a {{GPUSize64}} |size|
-      the following validation rules apply:
+        1. Throw an {{OperationError}} and stop.
 
-      1. |this|.{{GPUBuffer/[[state]]}} must be [=buffer state/mapped=] or [=buffer state/mapped at creation=].
-      1. |offset| must be a multiple of 8.
-      1. |size| must be a multiple of 4.
-      1. |offset| must be greater than or equal to |this|.{{[[mapping_range]]}}[0].
-      1. |offset| + |size| must be less than or equal to |this|.{{[[mapping_range]]}}[1].
-      1. [|offset|, |offset| + |size|) must not overlap another range in |this|.{{[[mapped_ranges]]}}.
+    1. Let |m| be a new {{ArrayBuffer}} of size |rangeSize| pointing at the content
+        of |this|.{{[[mapping]]}} at offset |offset| - |this|.{{[[mapping_range]]}}[0].
```

Comment:
We agreed on the call that `offset` should be relative to the start of the buffer, not to the active range.
So this should be "at offset |offset|"

Reply:
That's what this line says. In the above line we specify the offset w.r.t `this.[[mapping]]`, therefore I used the above words.

Would it be better to write 
```
pointing at the content of |this| at offset |offset|
```
?

Reply:
oh, I think I understand it better now